src/test/ui/async-await/issue-64130-4-async-move.rs

```diff
@@ -10,16 +10,15 @@ impl Client {
     }
 }
 
-async fn get() { }
+async fn get() {}
 
 pub fn foo() -> impl Future + Send {
     //~^ ERROR future cannot be sent between threads safely
-    let client = Client(Box::new(true));
-    async move {
-        match client.status() {
+    async {
+        match Client(Box::new(true)).status() {
```

Comment:
Is this just cleaning up the test, or were changes needed so that it still tested the same behavior as before? In other words, did the error from the previous version go away due to your changes?

Reply:
The error goes away with this change. However, I checked the issue #64130 and it seems that this test is intended for prompting hints for remedies when values living through yield points make generators non-`Send` or non-`Sync`. Therefore, I figured that it is good to keep this test by reproducing the previous errors.

Reply:
Sounds good!